database/pgsql/vulnerability.go

```diff
@@ -28,6 +28,50 @@ import (
 	"github.com/guregu/null/zero"
 )
 
+func (pgSQL *pgSQL) ListVulnerabilities(namespaceName string, limit int, page int) ([]database.Vulnerability, error) {
+	return listVulnerabilities(pgSQL, namespaceName, limit, page)
+}
+
+func listVulnerabilities(queryer Queryer, namespaceName string, limit int, page int) ([]database.Vulnerability, error) {
```

Comment:
This can be a single function. This pattern with the `Queryer` parameter is used in `scanVulnerability` because one of the caller needs to use a transaction, while another caller doesn't.